front/app/hooks/useProjectAllowedInputTopics.ts

```diff
@@ -11,12 +11,11 @@ export type IProjectAllowedInputTopicsState =
   | IProjectAllowedInputTopic[]
   | NilOrError;
 
-export default function useProjectAllowedInputTopics(projectId?: string) {
+export default function useProjectAllowedInputTopics(projectId: string) {
```

Comment:
Making this required as we pass it everywhere we use useProjectAllowedInputTopics. Would only make it optional if we need it for some reason.